README.md

```diff
@@ -36,7 +38,7 @@ $ source /cvmfs/dune.opensciencegrid.org/products/dune/setup_dune.sh
 $ ups list -aK+ dune-plot-style
 
 # set up a specific version
-setup dune-plot-style v00.02
+setup dune-plot-style v00_02
```

Comment:
I also wonder if we want to have the latest version setup by default - you can tag versions as `current` in ups. Can't think of many use cases we'd _not_ want the latest.

Reply:
Ah!  Yes.  Can you make sure that happens in the UPS tagging script?

Reply:
@admlw ?

Reply:
Sorry, yes. I don't think this needs to go into the script itself, they just need to use a couple of options when transferring to cvmfs. I can double check I'm remembering that and follow up with Jake about it today.

Reply:
Ok.  If that turns out to be the case, let me know, and I'll merge this PR.

Reply:
confirmed!